explainability-service/src/test/java/org/kie/trustyai/service/endpoints/metrics/EndpointTestProfile.java

```diff
@@ -0,0 +1,19 @@
+package org.kie.trustyai.service.endpoints.metrics;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+public class EndpointTestProfile implements QuarkusTestProfile {
+    @Override
+    public Map<String, String> getConfigOverrides() {
+        final Map<String, String> overrides = new HashMap<>();
+        overrides.put("service.storage-format", "PVC");
+        overrides.put("service.model-name", "example");
+        overrides.put("service.data-format", "CSV");
+        overrides.put("service.metrics-schedule", "5s");
```

Comment:
We can probably need to disable scheduler in a future PR in tests but for now it is fine